modules/installation-guide/partials/con_workspace-exposure-strategies.adoc

```diff
@@ -34,13 +33,26 @@ When any of those single-host types is used, all workspaces are deployed to sub-
 
 This is convenient for TLS-secured {prod-short} servers because it is sufficient to have a single certificate for the {prod-short} server, which will cover all the workspace component deployments as well.
 
-This strategy limits the exposed components and user applications. Any absolute URL generated on the server side that points back to the server does not work. This is because the server is hidden behind a path-rewriting Ingress that hides the workspace and the component-specific URL prefix from the server.
+There are two types of endpoints, that are defined in the devfiles, exposures in single-host strategy. To configured it, use `++CHE_INFRA_KUBERNETES_SINGLEHOST_WORKSPACE_DEVFILE__ENDPOINT__EXPOSURE++`. It is effective only in single-host server strategy..
```

Comment:
I'm not sure if English uses the "implanted" sentences the same way as Czech. For me, the English version is hard to follow unless I translate it into Czech at which point it suddenly makes much more sense :)

What about rewording it as:

There are 2 ways of exposing the endpoints specified in the devfile. These can be configured using the `CHE_INFRA_KUBERNETES_SINGLEHOST_WORKSPACE_DEVFILE__ENDPOINT__EXPOSURE` environment variable of the `{prod-short}`. This environment variable is only effective with the single-host server strategy and is applicable to all workspaces of all users.

Reply:
yes, fixed